target/ppc/translate/fixedpoint-impl.c.inc

```diff
@@ -111,6 +168,37 @@ TRANS64(LDU, do_ldst_D, true, false, MO_Q)
 TRANS64(LDUX, do_ldst_X, true, false, MO_Q)
 TRANS64(PLD, do_ldst_PLS_D, false, false, MO_Q)
 
+/* Load Quadword */
+static bool trans_LQ(DisasContext *ctx, arg_D *a)
+{
+    REQUIRE_INSNS_FLAGS(ctx, 64BX);
+
+    if (unlikely(a->rt & 1) || a->ra == a->rt) {
```

Comment:
shouldn't unlikely circle all conditions for the invalid instruction exception?

Reply:
That's a good question. This code comes from the legacy implementation, I'll `git blame` it to see if this was discussed before.

Reply:
This became a little less relevant now that the (a->rt & 1) check was moved to decodetree. You might still want to add the unlikely as ``if (unlikely(a->ra == a->rt) && !store) {``, though, but I don't care about it that much. After deciding, feel free to mark this as resolved.

Reply:
Fixed in https://github.com/PPC64/qemu/pull/14/commits/79340de493e92f1a923ab49c4c98c5c3cbaa5a3e